### ISSUE TITLE
Stewart: Images and Other Media Bleed Over Edge

### ISSUE THREAD

**Issue by macmanx2** [Automattic/themes#5652]
## Steps to replicate

View any post with an image or embedded video on a mobile device or simulated mobile device via browser inspector.

## Result

The image and/or embedded video will bleed off the edge to the right, requiring horizontal scroll. You can see this in action on the theme's demo at https://wp-themes.com/stewart/?p=19

## Expected

Images and embedded media should be scaled to fit the device's width.

* User report: self-reported

**Comment by kjellr** [comment]
This appears to be the same issue noted here: https://github.com/WordPress/gutenberg/issues/38384

I'm not sure if that the fix has made it to the theme demo there, but we should check if we can reproduce running the latest Gutenberg version.

**Comment by macmanx2** [comment]
Hm, it sounds similar, but in my own testing, this was not limited to the Classic editor. It affected image and Youtube blocks as well.

I'm AFK, so I'm unable to get a test site spun up to see if the latest Gutenberg plugin fixes it.

**Comment by kjellr** [comment]
Ok, thanks. I was unable to reproduce locally or on WP.com, but I'll tag this for the dev team to investigate as well.

**Comment by macmanx2** [comment]
Well, I did some more digging, and I guess we'll have to call it closed. I'm not sure what was going on (cache, maybe?) but it's fixed now.

Here's a screenshot of https://macmanx.com/2020/08/21/rewatch-highlander-the-series/ pre-fix that shows the issue on an iPhone 13 running iOS 15.3.1:

![IMG_7334](https://user-images.githubusercontent.com/4856235/158033700-d52f8798-cde1-4fdd-bb68-7d7002a0913e.jpeg)

I noticed some other posts with YouTube embeds were just fine, so I edited both to compare, and settings were exactly the same in each YouTube block, and there were also no notable differences between the posts.

In the process, I removed the offending YouTube block, re-added it, and updated the problem post, which somehow fixed it:

![IMG_7335](https://user-images.githubusercontent.com/4856235/158033740-8f349151-fdef-4394-916a-7253bb96736a.jpeg)

So ... 🤷‍♂️ again, there were no differences in the blocks, and no notable differences in the posts, but removing and re-adding (and again changing nothing) fixed it.

I guess I'll mark this as close then. Thanks for confirming you didn't see it @kjellr! It prompted me to look deeper. :)

![cache](https://user-images.githubusercontent.com/4856235/158033781-b7fb5821-535a-445a-8fc6-63dd7bb34270.jpg)
